flixel/system/FlxBasePreloader.hx

```diff
@@ -109,8 +111,8 @@ class FlxBasePreloader extends NMEPreloader
 	 */
 	private function onEnterFrame(E:Event):Void
 	{
-		var time:Int = Lib.getTimer();
-		var min:Int = Std.int(minDisplayTime * 1000);
+		var time = Date.now().getTime() - _startTime;
```

Comment:
Why use `Date.now()` instead of  `Lib.getTimer()` - does the latter not work in HTML5?